tst/EnergyPlus/unit/Furnaces.unit.cc

```diff
@@ -1179,4 +1179,40 @@ TEST_F(EnergyPlusFixture, UnitaryHeatPumpAirToAir_MaxSuppAirTempTest)
     EXPECT_NEAR(45.0, state->dataLoopNodes->Node(state->dataFurnaces->Furnace(1).FurnaceOutletNodeNum).Temp, 0.000001);
 }
 
+TEST_F(EnergyPlusFixture, Furnaces_SetMinOATCompressor)
+{
+    // Test for Issue 9121
+
+    state->dataFurnaces->Furnace.allocate(1);
+    state->dataFurnaces->Furnace(1).CoolingCoilType_Num = Coil_CoolingAirToAirVariableSpeed;
+
+    state->dataVariableSpeedCoils->GetCoilsInputFlag = false;
+
+    state->dataVariableSpeedCoils->VarSpeedCoil.allocate(1);
+    state->dataVariableSpeedCoils->VarSpeedCoil(1).MinOATCompressor = 30.0;
+
+    state->dataFurnaces->Furnace(1).HeatingCoilType_Num = CoilVRF_Heating;
```

Comment:
why is there a vrf heating coil in a furnace?

Reply:
This is just to trick the code to not execute any of the `if`s for the heating branch, as there is no problem there--so just randomly picked any heating coil type other than Coil_HeatingAirToAirVariableSpeed or CoilDX_HeatingEmpirical, to save one unnecessary function call (VariableSpeedCoils::GetVSCoilMinOATCompressorUsingIndex() orDXCoils::GetMinOATCompressorUsingIndex()).

Reply:
I should not merge this because it's an ill formed unit test. I pulled your repo and used this and it worked fine:

    state->dataFurnaces->Furnace(1).HeatingCoilType_Num = Coil_HeatingElectric;

You want me to push this? I will merge this branch with this update.

Reply:
@rraustad Thanks! That change looks pretty good!